pyhealth/datasets/base_dataset.py

```diff
@@ -1,15 +1,14 @@
 import logging
-import os
 from abc import ABC, abstractmethod
 from collections import Counter
 from copy import deepcopy
-from typing import Optional, List, Dict, Callable, Tuple, Union
+from typing import Dict, Callable, Tuple, Union
 
 from torch.utils.data import Dataset
 from tqdm import tqdm
 
 from pyhealth.data import Patient, Event
-from pyhealth.datasets.utils import MODULE_CACHE_PATH, hash_str
```

Comment:
It is still good to explicitly write down what we are importing, otherwise, we may have to look into `pyhealth.datasets.utils` and check what tools are available.

Reply:
resolved in latest commit